cmd/libsnap-confine-private/string-utils-test.c

```diff
@@ -74,6 +74,25 @@ static void test_sc_startswith(void)
 	g_assert_false(sc_startswith("ba", "bar"));
 }
 
+static void test_sc_is_beneath(void)
```

Comment:
Note: @mvo5 this code was removed in the original branch as it contains some contradictions. Please make sure it is not resurrected once this is merged back to master.